src/components/policy/policy_regular/src/policy_table/validation.cc

```diff
@@ -114,14 +114,46 @@ bool ApplicationPoliciesSection::Validate() const {
   return true;
 }
 
-bool ApplicationParams::Validate() const {
-  // Check for empty "groups" sub-sections
-  if (groups.empty()) {
```

Comment:
This check for groups.empty() was not replaced below.

Reply:
@JackLivio  why groups can't be empty?

Reply:
@JackLivio goups has type : 
```typedef Array<String<1, 255>, 0, 255> Strings;```
With minimal value - 1, so there no need for additional check of empty groups. It will be done implicitly.

Reply:
@JackLivio  why groups should not be empty?

Reply:
@LuxoftAKutsan Not sure, just saying that it was the logic that was implemented before.

Reply:
@JackLivio  suppose this regression was discovered during ATF scripts run. Looks like empty groups are allowed in application params in policy table.
@OlesiaV  can you approve?